std_msgs/Header header

int32 id
geometry_msgs/Pose wall_center
PlaneData[] x_planes
PlaneData[] y_planes
